frontend/app/views/event/masterclassesList.scala.html

```diff
@@ -19,10 +19,9 @@
     <main>
 
         <div class="l-constrained">
-        @fragments.event.headerBarWithLinks(
-            twitterLink="https://twitter.com/guardianlive",
-            showFormLink=true
-        )
+            <section class="header-bar">
+                <h1 class="header-bar__title">Guardian Masterclasses offers a range of practical, expert-led online workshops in journalism, creative writing, business skills and personal development. To be the first to hear about new events, <a href="https://www.theguardian.com/guardian-masterclasses/2015/jan/19/sign-up-to-the-guardian-masterclasses-newsletter">sign up to our newsletter here</a>.</h1>
+            </section>            
```

Comment:
We decided to inline this because it's only used in one place. This also means that headBarWithLinks is no longer required so we've deleted it.